src/modals/AddItemModal.tsx

```diff
@@ -32,17 +32,16 @@ const AddItemModal: FC<AddItemModalProps> = ({ tooltipLabel, onRefresh }) => {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const finalRef = React.useRef(null);
   const toast = useToast();
+
+  const itemRef = useRef<HTMLInputElement>(null);
+
   const [itemData, setItemData] = useState({
-    name: "",
+    name: itemRef.current?.value,
```

Comment:
Initial state should be empty
```suggestion
    name: "",
```